docs/reference/sql/endpoints/odbc/configuration.asciidoc

```diff
@@ -260,7 +262,110 @@ image:images/sql/odbc/env_var_log.png[]
+named "Connection string", "String extras", or similar).  The format of the
+string is `Attribute1=Value1`. Multiple attributes can be specified, separated
+by a semicolon `Attribute1=Value1;Attribute2=Value2;`. The attribute names are
+given below.
+
+===== Timeout
+
+The maximum time (in seconds) a request to the server can take. This can be
+overridden by a larger statement-level timeout setting. The value 0 means no
+timeout.
+
+Default: `0`.
+
+===== Follow
+
+A boolean value (`yes`|`no` / `true`|`false` / `0`|`1`) controlling if the
+driver will follow HTTP redirects.
+
+Default: `yes`.
+
+===== MaxFetchSize
+
+The maximum number of rows that {es-sql} server should send the driver for one
+page. This corresponds to {es-sql}'s request parameter `fetch_size`. The value
+0 means server default.
+
+Default: `0`.
+
+===== MaxBodySizeMB
+
+The maximum size (in megabytes) that an answer can grow to, before being
+rejected as too large by the driver.
+This is concerning the HTTP answer body of one page, not the cumulated data
+volume that a query might generate.
+
+Default: `100`.
+
+===== ApplyTZ
+
+A boolean value controlling the timezone of:
+
+* the context in which the query will execute (especially relevant for functions dealing with timestamp components);
+
+* the timestamps received from / sent to the server.
+
+If disabled, the UTC timezone will apply; otherwise, the local machine's set
+timezone.
+
+Default: `no`.
+
+===== ScientificFloats
+
+Controls how the floating point numbers will be printed, when these are
+converted to string by the driver. Possible values given to this parameter:
+
+* `scientific`: the exponential notation (ex.: 1.23E01);
+
+* `default`: the default notation (ex.: 12.3);
+
+* `auto`: the driver will chose one of the above depending on the value to be
+printed.
+
+Note that the number of decimals is dependent on the precision (or ODBC scale)
+of the value being printed and varies with the different floating point types
+supported by {es-sql}.
+This setting is not effective when the application fetches from the driver the
+values as numbers itself and then does the conversion subsequently.
+
+Default: `default`.
+
+===== VersionChecking
+
+By default, the version of the driver and that of the server must be the same.
+This parameter will allow a driver to connect to a server of different version.
+The variation however is only at minor version, both endpoints must be of same version number.
+
+WARNING: This parameter can only be used for troubleshooting purposes. Running
+with versions out of sync is not supported.
+
+Possible values:
+
+* `strict`: the versions must be in sync;
+
+* `major`: the versions must have the same major number.
+
+Default: `strict`.
+
+===== MultiFieldLenient
+
+This boolean parameter controls the behavior of the server in case a
+multi-value field is queried. In case this is set and the server encounters
+such a field, it will pick the first value in the set and return it as the
```

Comment:
I would add the same wording used for the REST API (I'm working on adding a note for JDBC, as well) about what "first value" means:
> (without any guarantees of what that will be - typically the first in natural ascending order)

From https://www.elastic.co/guide/en/elasticsearch/reference/master/sql-rest.html.

Reply:
Good point, thanks. I've also added the cross ref to paragraph with the parameters description.